Backend/src/modules/users/users.service.ts

```diff
@@ -37,4 +52,25 @@ export default class UserService {
     });
     return user;
   }
+
+  public async updateUser(userData: UpdateUserInfo): Promise<User> {
+    const { id, data } = userData;
+
+    const updatedUser = await prisma.user.update({
+      where: { id },
+      data: {
+        firstName: data.firstName,
```

Comment:
Again, how you are going to handle partial update? One solution is to retrieve user and set default values.